docs/api.rst

```diff
@@ -25,7 +25,8 @@ Response:
 
 Get flashCards
 **************
-`</api/v1/flashcard/deck/\<deck:id\>>`_ 
```

Comment:
this line showed error syntax and i dont find solutionto use urls in generic style